slave.mk

```diff
@@ -648,6 +653,11 @@ $(foreach IMAGE,$(SONIC_STRETCH_DOCKERS), $(eval $(IMAGE)_FILES_PATH := $(STRETC
 $(foreach IMAGE,$(SONIC_STRETCH_DBG_DOCKERS), $(eval $(IMAGE)_DEBS_PATH := $(STRETCH_DEBS_PATH)))
 $(foreach IMAGE,$(SONIC_STRETCH_DBG_DOCKERS), $(eval $(IMAGE)_FILES_PATH := $(STRETCH_FILES_PATH)))
 
+$(foreach IMAGE,$(SONIC_BUSTER_DOCKERS), $(eval $(IMAGE)_DEBS_PATH := $(BUSTER_DEBS_PATH)))
+$(foreach IMAGE,$(SONIC_BUSTER_DOCKERS), $(eval $(IMAGE)_FILES_PATH := $(BUSTER_FILES_PATH)))
+$(foreach IMAGE,$(SONIC_BUSTER_DBG_DOCKERS), $(eval $(IMAGE)_DEBS_PATH := $(BUSTER_DEBS_PATH)))
+$(foreach IMAGE,$(SONIC_BUSTER_DBG_DOCKERS), $(eval $(IMAGE)_FILES_PATH := $(BUSTER_FILES_PATH)))
```

Comment:
do not think this is needed.

Reply:
Let me try without this and see how it goes.